polaris-react/src/styles/_tokens.scss

```diff
@@ -0,0 +1,32 @@
+/* stylelint-disable unit-disallowed-list */
```

Comment:
TODO:
- Do NOT commit this file - Add to `.gitignore`
- Instead generate it in a `prebuild` script

Reply:
Done